src/components/print/PrintLayerService.js

```diff
@@ -89,8 +88,7 @@ goog.require('ga_urlutils_service');
           'type': 'Vector',
           'styles': {
             '1': { // Style for marker position
-              'externalGraphic':
-                gaUrlUtils.nonCloudFrontUrl(options.markerUrl),
+              'externalGraphic': options.markerUrl,
```

Comment:
How did this even work before? The nonCloudFrontUrl function doesn't exist, does it?